### PR TITLE
mgmt, local generation for eventhub

### PR DESCRIPTION
**Pull request by v-hongli1**
# Description

No manual code.

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 2.27.0-beta.1 (2023-04-23)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Updated core dependency from resources.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md:9]: Change log would be like "- Updated `api-version` to `2021-11-01`."

File: sdk/resourcemanager/azure-resourcemanager-monitor/pom.xml [hunk 2]
```diff
@@ -102,7 +102,7 @@
     <dependency>
       <groupId>com.azure.resourcemanager</groupId>
       <artifactId>azure-resourcemanager-eventhubs</artifactId>
-      <version>2.26.0</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-eventhubs;current} -->
+      <version>2.27.0-beta.1</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-eventhubs;current} -->
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-monitor/pom.xml:105]: Why do we want to change the monitor version?

File: sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md (hunk 1 above)
Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md:9]: Fixed in the new version

File: sdk/resourcemanager/azure-resourcemanager-monitor/pom.xml (hunk 2 above)
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-monitor/pom.xml:105]: Seems no problem, as eventhubs gets to 2.27.0-beta.1

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-monitor/pom.xml:105]: `eventhub` is referenced in `monitor`, so when executing the `Prepare release` step of `Preview release for premium`, the version of this reference has changed. Do you want to discard this?

File: sdk/resourcemanager/azure-resourcemanager-eventhubs/pom.xml [hunk 3]
```diff
@@ -37,13 +37,14 @@
...
-    <jacoco.min.linecoverage>0.10</jacoco.min.linecoverage>
-    <jacoco.min.branchcoverage>0.10</jacoco.min.branchcoverage>
+    <jacoco.min.linecoverage>0</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0</jacoco.min.branchcoverage>
     <!-- Configures the Java 9+ run to perform the required module exports, opens, and reads that are necessary for testing but shouldn't be part of the module-info. -->
     <javaModulesSurefireArgLine>
       --add-opens com.azure.resourcemanager.eventhubs/com.azure.resourcemanager.eventhubs=ALL-UNNAMED
       --add-opens com.azure.resourcemanager.storage/com.azure.resourcemanager.storage=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-eventhubs/pom.xml:47]: Wonder what breaks here...

File: sdk/resourcemanager/azure-resourcemanager-monitor/pom.xml (hunk 2 above)
Comment (XiaofeiCao) [reply sdk/resourcemanager/azure-resourcemanager-monitor/pom.xml:105]: Got it. LGTM.

File: sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md]: Please, what api-version do you upgrade here?

File: sdk/resourcemanager/azure-resourcemanager-eventhubs/pom.xml (hunk 3 above)
Comment (XiaofeiCao) [reply sdk/resourcemanager/azure-resourcemanager-eventhubs/pom.xml]: OK, I saw the breaks are about `Indentity` classes. They are due to our recent codegen fix. We can skip the revapi.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2725264&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=68a849d6-f425-5f38-d450-7e2c3d1df885

File: sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 2.27.0-beta.1 (2023-04-23)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Updated `api-version` to `2.27.0-beta.1`.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md:9]: still not correct.

OK, make sure you understand what is api-verion.

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/eventhub/resource-manager#tag-package-2022-10-preview

On path to swagger, as well as swagger info, there is api-version
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-10-01-preview/eventhubs.json#L6
that follow `year-month-day` or `year-month-day-preview`

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md (hunk 4 above)
Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md]: I see, now fixed in the new version.

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**